pkg/apis/serving/v1alpha1/kfservice_framework.go

```diff
@@ -32,15 +32,19 @@ const (
 	ExactlyOneModelSpecViolatedError = "Exactly one of [Custom, Tensorflow, SKLearn, XGBoost] must be specified in ModelSpec"
 	// AtLeastOneModelSpecViolatedError is a known error message
 	AtLeastOneModelSpecViolatedError = "At least one of [Custom, Tensorflow, SKLearn, XGBoost] must be specified in ModelSpec"
+
+	TensorflowServingImageConfigName = "TensorflowServingImage"
+	XgboostServingImageConfigName    = "XgboostServingImage"
+	SklearnServingImageConfigName    = "SklearnServingImage"
```

Comment:
Can we keep the keys here consistent with actual image names we choose?

Reply:
Done